src/main/java/Ebot.java

```diff
@@ -0,0 +1,156 @@
+import java.util.Scanner;
+
+public class Ebot {
+    private static final String lineDivider = ("___________________________________________");
+    private static final String bigIndent = "\t\t";
+    private static final String smallIndent = "\t";
+
+    public static void main(String[] args) {
+        String welcomeMessage = (smallIndent + "Hello! I'm Ebot" +
+                System.lineSeparator() +
+                smallIndent + "What can i do for you?");
+        String exitMessage = (smallIndent + "Bye. Hope to see you again soon!");
+        System.out.println(lineDivider + System.lineSeparator() + welcomeMessage);
+        System.out.println(lineDivider);
+
+        // Program start, initialize scanner for user inputs
+        String userInput;
+        Scanner in = new Scanner(System.in);
+        userInput = "";
+
+        // Initialize list
+        int numberOfLists = TaskList.getNumberOfTaskLists();
+        String taskListName = ("list") + numberOfLists;
+        TaskList taskList = new TaskList(taskListName);
+
+        while (!userInput.equalsIgnoreCase("bye")) {
+            userInput = in.nextLine();
+            // if user types "bye" command
+            if (userInput.equalsIgnoreCase("bye")) {
+                System.out.println(exitMessage);
+                System.out.println(lineDivider);
+
+            // if user types "list" command
+            } else if(userInput.equalsIgnoreCase("list")) {
```

Comment:
Can consider adding a space between `if` and the parentheses
```suggestion
            } else if (userInput.equalsIgnoreCase("list")) {
```